### DISCUSSION TITLE
Velocity and droplet distribution within the water spray

### DISCUSSION THREAD

**Discussion by hassanrezavi** [firemodels/fds#13147] [Q&A]
When I am doing the experiments on the water spray, I get the values of the water droplets with laser analysis as follows, the Dv0.5 sizes were found at the following locations below the nozzle: 28 µm (25 mm), 35 µm (50 mm), 40 µm (100 mm), 45 µm (150 mm), and 48 µm (300 and 500 mm).
How can we simulate this case in FDS ? Help !

**Comment by drjfloyd** [comment]
First step make sure you have read the FDS Technical Reference and User's Guide. Then there are many many examples of sprinkler inputs in the User's Guide, Verificaiton Guide, and the Validation Guide.

**Comment by mcgratta** [comment]
The validation cases `BRE Sprays` and `VTT Sprays` might help. Search in the FDS Validation Guide.
